docs/versions/latest/Release.md

```diff
@@ -1,128 +1,24 @@
 # (VERSION_PLACEHOLDER DO NOT DELETE)
-
 [//]: # (Used when working on a new release. Placed together with the Version.md)
 [//]: # (Nothing here is optional. If a step must not be performed, it must be said so)
 [//]: # (Do not fill the version, it will be done automatically)
 [//]: # (Quick Intro to what is the focus of this release)
 
 ## Breaking Changes
-
 [//]: # (### *Breaking Change*)
 [//]: # (Describe the breaking change AND explain how to resolve it)
 [//]: # (You can utilize internal links /e.g. link to the upgrade procedure, link to the improvement|deprecation that introduced this/)
 
-## Deprecations
 
+## Deprecations
 [//]: # (### *Deprecation*)
 [//]: # (Explain what is deprecated and suggest alternatives)
 
-[//]: # (Features -> New Functionality)
 
+[//]: # (Features -> New Functionality)
 ## Features
-
```

Comment:
The following code should not be deleted as it describes several new features of the vRBT

Reply:
Reverted.